.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -29,9 +29,19 @@ assignees: ''
     - publishes the docker `latest` tag variants
 - [ ] Check binary SHAs are correct on the release page
 - [ ] Check "Container Verify" GitHub workflow has run successfully
-- [ ] Create besu-docs release - https://github.com/hyperledger/besu-docs/releases/new
-  - Copy release notes from besu
-  - If publishing the release in github doesn't automatically trigger this workflow, then manually run https://github.com/hyperledger/besu-docs/actions/workflows/update-version.yml
-- [ ] Create homebrew release - run GHA workflow directly https://github.com/hyperledger/homebrew-besu/actions/workflows/update-version.yml
+- [ ] Create besu-docs release
+  - Run the GitHub workflow [update-version.yml](https://github.com/hyperledger/besu-docs/actions/workflows/update-version.yml) to update the docs version
+    - This creates a branch with name `besu-version-<version>`. PR may also be created if action has the permission
+    - If the PR has not been automatically created, create the PR manually to merge the branch to main
+    - Verify new version has been added as the stable version and last version
+    - New folder created `versioned_docs/version-<version>` [example pull request](https://github.com/hyperledger/besu-docs/pull/1675/files)
+    - Review and merge the PR to main
+  - Create a new release for the version. Copy the release notes from [hyperledger/besu](https://github.com/hyperledger/besu) to the besu-docs release
+- [ ] Create homebrew release
+  - Run GHA workflow directly [update-version.yml](https://github.com/hyperledger/homebrew-besu/actions/workflows/update-version.yml)
+  - This GHA workflow will create a branch with name `update-<version>`. Workflow may create the PR if permission allowed
+  - If the PR has not been automatically created, create the PR manually to merge the branch to main
+  - Review and merge the PR to main
+  - Run commands `brew tap hyperledger/besu && brew install besu` on MacOSX and verify latest version has been installed
```

Comment:
Did we discuss automating this step as well - is that still the plan?

Reply:
It is in the plan and require security review from Hyperledger as well. In order to trigger this workflow from Besu release it requires a GitHub app created in the GitHub org hyperledger and have necessary permissions. GitHub organisation app need to be created by a organisations owner. I am planning to draft the architecture and validate that in a test GitHub organisation and propose the solution. If the solution is accepted and Hyperledger GitHub admins happy with the security we can proceed to implement it.